processingreport.qmd

```diff
@@ -0,0 +1,41 @@
+---
+title: "U.S. Greenhouse Gas Center: Processing and Verification Reports"
+---
+
+Welcome to the [U.S. Greenhouse Gas (GHG) Center](https://earth.gov/ghgcenter) processing and verification reports. These reports verify that the accuracy and integrity of each dataset in the US GHG Center is maintained once it is processed into the Center.
+
+The reports are grouped topically and labeled by dataset name. The dataset product type (model output, satellite observation, etc.) is noted next to each. Click on a dataset name to view the processing and verification report for that dataset.
+
+Examples of processing that may occur include transforming data from its source format into a could-optimized format, converting the units of the source data into a more common or standard unit, and flagging "nodata" values to ensure accurate data visualization. We strive to handle all data with extreme care, and share these reports to provide transparency and insight into any processing that is applied, while ensuring accuracy and reliability every step of the way.
+
+Join us in our mission to make data-driven environmental solutions accessible.
+Explore, analyze, and make a difference with the US GHG Center.
+
+[View the US GHG Center Data Catalog](https://earth.gov/ghgcenter/data-catalog)
+
+## Gridded Anthropogenic Greenhouse Gas Emissions
+1. [OCO-2 MIP Top-Down CO₂ Budgets](processing_and_verification_reports/oco2-mip-co2budget-yeargrid-v1_Processing and Verification Report.qmd) - Model Output
```

Comment:
Consider adding 'processing and verification report' after each
E.g. "OCO-2 MIP Top-Down CO₂ Budgets Processing and Verification Report"

Reply:
and remove the product type "- Model Output"